plugin/trino-iceberg/pom.xml

```diff
@@ -152,6 +152,12 @@
             <groupId>org.apache.iceberg</groupId>
             <artifactId>iceberg-api</artifactId>
             <version>${dep.iceberg.version}</version>
+            <exclusions>
```

Comment:
Is this due to a version conflict?

Reply:
Yes, it appears there is a patch-level version dependency drift and the dependency enforcer caught it.

Reply:
I'm not sure if there's a better way to resolve this.  This was the error (repeated for all iceberg dependencies):

```
Require upper bound dependencies error for org.slf4j:slf4j-api:1.7.32 paths to dependency are:
+-io.trino:trino-iceberg:391-SNAPSHOT
  +-org.apache.iceberg:iceberg-api:0.14.0
    +-org.slf4j:slf4j-api:1.7.32 (managed) <-- org.slf4j:slf4j-api:1.7.25
```

Iceberg is currently on `1.7.25`

Reply:
That's weird that it complains when a dependency has an older patch version. Note that it would be good to update Iceberg to the latest 1.7.36, since some of the SLF4J components have CVEs in 1.7.25 and stupid security scanners will flag the API JAR (even though it's not affected).

Reply:
We can definitely update in the next release (I'll create a PR for updating).  There was a gradle update and it may have changed how these are declared which triggered the enforcer, but the dependency is valid and hasn't changed.